public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -91,14 +91,26 @@ export class PanelHeader extends Component<Props, State> {
     this.props.panel.getQueryRunner().cancelQuery();
   };
 
-  private renderLoadingState(): JSX.Element {
-    return (
-      <div className="panel-loading" onClick={this.onCancelQuery}>
-        <Tooltip content="Cancel query">
-          <Icon className="panel-loading__spinner spin-clockwise" name="sync" />
-        </Tooltip>
-      </div>
-    );
+  private renderLoadingState(state: LoadingState): JSX.Element {
+    if (state === LoadingState.Loading) {
+      return (
+        <div className="panel-loading" onClick={this.onCancelQuery}>
+          <Tooltip content="Cancel query">
+            <Icon className="panel-loading__spinner spin-clockwise" name="sync" />
+          </Tooltip>
+        </div>
+      );
+    }
+    if (state === LoadingState.Streaming) {
+      return (
+        <div className="panel-loading" onClick={this.onCancelQuery}>
+          <Tooltip content="Streaming (unsubscribe)">
+            <Icon name="circle" />
```

Comment:
You can use already implemented [monochrome circle icon](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Icon/assets/Circle.tsx). 
![image](https://user-images.githubusercontent.com/30407135/93770013-edabd780-fc1b-11ea-9d08-f58bac2d4b23.png)

So you just need to add `type="mono"` and use css (className/style) to update its color to green.